### ISSUE TITLE
Build failure with `'O=O=test' violates pattern constraint `

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno.resizetizer#299]
## Current behavior

The default template sets ApplicationPublisher to:

```
O=Something
```

This fails with:

```
DEP0700: Registration of the app failed. [0x80080204] error 0xC00CE169: App manifest validation error: 
The app manifest must be valid as per schema: 
Line 32, Column 44, Reason: 'O=O=test' violates pattern constraint of 
'(CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|Description|PostalCode|POBox|Phone|X21Address|dnQualifier|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")(, ((CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|Description|PostalCode|POBox|Phone|X21Address|dnQualifier|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")))*'.
```

**Comment by dansiegel** [comment]
This is sounding like an issue with the template Wizard. I didn't see this when testing with the CLI template. 

I believe we changed the way the Publisher is handled when we started updating the appxmanifest. That should be good in the template but probably needs to be fixed in the Wizard if you're seeing this. 

Good to have some sanity checks for Resizetizer in the future though.

**Comment by jeromelaban** [comment]
Fixed in vsix 5.3.8
